quick_start/sample_app.py

```diff
@@ -0,0 +1,350 @@
+import enum
+import json
+import uuid
+from dataclasses import dataclass
+import inflection as inflection
+from azure.kusto.ingest import QueuedIngestClient
+from quick_start.utils import Utils, AuthenticationModeOptions
```

Comment:
This requires the package to be installed, replace with:
`from utils import Utils, AuthenticationModeOptions`